src/actions/about.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { getDashboardVersion, getPipelineVersion } from '../api';
+
+export async function fetchDashboardInfo(namespace) {
```

Comment:
I'm pretty sure this isn't actually an action, so for example notice async dispatch usage at https://github.com/tektoncd/dashboard/blob/master/src/actions/secrets.js, this is more like an API function for comms or index

Reply:
Have moved this function to index